### PR TITLE
qbd-direct integrations configurations page setting

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

File: src/app/branding/c1-contents-config.ts [hunk 1]
```diff
@@ -4,6 +4,91 @@ import config from './config.json';
 export const brandingConfig: BrandingConfiguration = config as BrandingConfiguration;
 
 export const c1Contents = {
+    qbd_direct: {
+        landing: {
+            contentText: 'Import chart of accounts and projects from NetSuite and export expenses from your Expense Management account.',
```
Comment (ashwin1111) [inline src/app/branding/c1-contents-config.ts:9]: replace all "NetSuite" with "QuickBooks Desktop"

File: src/app/branding/c1-contents-config.ts [hunk 2]
```diff
@@ -4,6 +4,91 @@ import config from './config.json';
...
+                stepName: 'Connect to NetSuite',
+                subLabel: 'Provide your credentials to establish a secure connection between your Expense Management and NetSuite account'
+            },
+            exportSetting: {
+                stepName: 'Export settings',
+                headerText: ' Export Corporate Card Expenses',
+                contentText: 'Configure how and when expenses from Expense Management can be exported to NetSuite.',
+                corporateCard: {
+                    creditCardExpenseLabel: 'How should the expenses be exported?',
+                    cccExpenseBankAccountSubLabel: 'The selected expense payment type will be added to the corporate credit card expenses exported from ' + brandingConfig.brandName + ' to NetSuite.',
```
Comment (ashwin1111) [inline src/app/branding/c1-contents-config.ts:25]: these are not fixed yet - let's just have default keys and anything accounting side specific don't add any content for now. Later is becomes pain to remove things.

File: src/app/branding/c1-contents-config.ts [hunk 3]
```diff
@@ -4,6 +4,91 @@ import config from './config.json';
...
+                    creditCardExpenseStateLabel: 'How should expenses be labeled  before exporting from Expense Management?',
+                    creditCardExpenseStateSubLabel: 'You can choose to only export expenses when they\'ve been labeled approved or closed.',
+                    defaultCCCAccountLabel: 'Set default credit card account as',
+                    defaultCCCAccountPlaceholder: 'Select default credit card account',
+                    defaultDebitCardAccountLabel: 'Set employee payables account as',
+                    defaultDebitCardAccountPlaceholder: 'Select default debit card account',
+                    defaultCCCVendorLabel: 'Set default corporate card vendor as',
+                    accountsPayableLabel: 'Set vendor payables account as',
+                    journalOptionLabel: 'Name in journal entry',
+                    journalOptionSubLabel: 'You can select either the \'merchant name\' or the \'employee name\' to appear in the \'name\' field of your journal entries.'
```
Comment (ashwin1111) [inline src/app/branding/c1-contents-config.ts:39]: 32-39

File: src/app/branding/c1-contents-config.ts [hunk 4]
```diff
@@ -4,6 +4,91 @@ import config from './config.json';
...
+                stepName: 'Import settings',
+                headerText: '',
+                contentText: 'Choose the required import fields from NetSuite to Expense Management.',
+                importCategoriesLabel: ' as categories',
+                importCategoriesSubLabel: ' will be available as categories in Expense Management.',
+                importCustomersLabel: 'Import customers from NetSuite',
+                importCustomersSubLabel: 'The customers in NetSuite will be imported as projects in ' + brandingConfig.brandName + ' and will be a selectable field while creating an expense',
+                taxCodeLabel: 'Import tax from NetSuite',
+                taxCodeSubLabel: 'The imported tax codes from NetSuite will be set as tax group in ',
+                defaultTaxCodeLabel: 'Select Default tax Code',
```
Comment (ashwin1111) [inline src/app/branding/c1-contents-config.ts:52]: 32-37

File: src/app/branding/c1-contents-config.ts [hunk 5]
```diff
@@ -4,6 +4,91 @@ import config from './config.json';
...
+                importSuppliersAsMerchantsLabel: 'Import vendors from NetSuite as merchants',
+                importSuppliersAsMerchantsSubLabel: 'The vendors in NetSuite will be imported as merchants in ' + brandingConfig.brandName + ' and will be a selectable field while creating an expense.',
```
Comment (ashwin1111) [inline src/app/branding/c1-contents-config.ts:54]: remove all accounting side related keys

File: src/app/branding/fyle-contents-config.ts [hunk 6]
```diff
@@ -4,6 +4,91 @@ import config from './config.json';
 export const brandingConfig: BrandingConfiguration = config as BrandingConfiguration;
 
 export const fyleContents = {
+    qbd_direct: {
```
Comment (ashwin1111) [inline src/app/branding/fyle-contents-config.ts:7]: same comments as above file

**Comment by DhaaraniCIT** [comment]
content file will edit when we reach the respective page

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**